scripts/zulip_emoji_merge_delegate.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+import sys
+import zulip
+import requests
+import re
+
+# Usage:
+# python scripts/zulip_emoji_merge_delegate.py $ZULIP_API_KEY $ZULIP_EMAIL $ZULIP_SITE $GITHUB_TOKEN
```

Comment:
Refer to the workflow file for an example how these variables look? Right now, reading just that comment would leave me pretty clue-less.